common/static/coffee/src/discussion/utils.coffee

```diff
@@ -113,7 +113,7 @@ class @DiscussionUtil
       alertDiv = $("<div class='modal' role='alertdialog' id='discussion-alert' aria-describedby='discussion-alert-message'/>").css("display", "none")
       alertDiv.html(
         "<div class='inner-wrapper discussion-alert-wrapper'>" +
-        "  <button class='close-modal dismiss' aria-hidden='true'><i class='icon fa fa-times'></i></button>" +
+        "  <button class='close-modal dismiss' title='Close modal'><span class='icon fa fa-times' aria-hidden='true'></span></button>" +
```

Comment:
Use gettext to translate "Close modal". And I'd recommend making it just "Close" or "Close dialog", as modal isn't a very user-friendly term.